include/lsst/afw/math/TransformBoundedField.h

```diff
@@ -0,0 +1,94 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2017 LSST Corporation.
```

Comment:
It should be different — not because of the date, but because the copyright isn't held by LSST Corporation. Our usual convention is to use "Copyright NNNN AURA/LSST" (which is likely also incorrect, but at least it's in the template...).